game-core/src/test/java/games/strategy/engine/lobby/client/ui/LobbyGameTableModelTest.java

```diff
@@ -50,13 +50,6 @@ void addGame() {
     assertThat(gameListModel.size(), is(2));
   }
 
-  @Test
-  void updateGameWithNullGuidIsIgnored() {
-    gameListModel.update(null, GameDescription.builder().build());
-    assertThat(
-        "expect row count to remain 1, null guid is bogus data", gameListModel.size(), is(1));
```

Comment:
Actually testing for an exception might be not that bad

Reply:
It's just testing then the precondition statement; I think that is at least questionable. It's not as questionable as testing a getter or setter, but I would lean towards considering that as testing more framework and the language rather than our code. 

If a method were more complex, that level of testing/scrutiny could be good safety, and I do have memories where certain illegal arg conditions were not detected properly. I don't think this is clearly a good case for it though